src/contrib/cluster/Akka.Cluster.Tools.Tests/Singleton/ClusterSingletonRestartSpec.cs

```diff
@@ -69,13 +67,9 @@ private void Join(ActorSystem from, ActorSystem to)
             {
                 AwaitAssert(() =>
                 {
-                    var m1 = Cluster.Get(from).State.Members;
-                    var sa = Cluster.Get(from).SelfUniqueAddress;
                     Cluster.Get(from).Join(Cluster.Get(to).SelfAddress);
-                    Cluster.Get(from).State.Members.Select(c => c.UniqueAddress)
-                        .Should().Contain(Cluster.Get(from).SelfUniqueAddress);
-                    Cluster.Get(from).State.Members.Select(c => c.Status)
-                        .Should().BeEquivalentTo(ImmutableHashSet.Create(MemberStatus.Up));
+                    Cluster.Get(from).State.Members.Select(c => c.UniqueAddress).Should().Contain(Cluster.Get(from).SelfUniqueAddress);
+                    Cluster.Get(from).State.Members.Select(x => x.Status).ToImmutableHashSet().Should().Equal(ImmutableHashSet.Create(MemberStatus.Up));
```

Comment:
@alexvaluyskiy yep, that's fine in this case. All we're doing is guaranteeing that all remaining nodes are up.